app/models/protocol.rb

```diff
@@ -49,6 +49,8 @@ class Protocol < ActiveRecord::Base
 
   has_many :principal_investigators, -> { where(project_roles: { role: %w(pi primary-pi) }) },
     source: :identity, through: :project_roles
+  has_many :authorized_users, -> { where.not(project_roles: { role: %w(pi primary-pi) }) },
```

Comment:
Isn't a PI technically an authorized user? I think this name could be a little confusing.